spec/dynamoid/persistence_spec.rb

```diff
@@ -63,6 +63,36 @@
     Address.table_name.should == 'dynamoid_tests_addresses'
   end
 
+  context 'no namespace defined' do
```

Comment:
Would you mind trying to factor some of these specs into a shared example group? You can use a let statement or a before block to redefine the namespace config value for each field. Check out the rspec docs and betterspecs.org if you haven't used shared groups before.